reference/latest/src/main/java/com/example/docs/item/armor/GuiditeArmorMaterial.java

```diff
@@ -0,0 +1,107 @@
+package com.example.docs.item.armor;
+
+import net.minecraft.item.ArmorItem;
+import net.minecraft.item.ArmorMaterial;
+import net.minecraft.recipe.Ingredient;
+import net.minecraft.sound.SoundEvent;
+import net.minecraft.sound.SoundEvents;
+
+import com.example.docs.item.ModItems;
+
+// :::1
+public class GuiditeArmorMaterial implements ArmorMaterial {
+	//:::1
+	// :::_10
+	public static final GuiditeArmorMaterial INSTANCE = new GuiditeArmorMaterial();
+	// :::_10
+	// :::1
+	// Base durability values for all the slots.
+	// Boots, Leggings, Chestplate, Helmet
+	// You shouldn't change these values.
+	private static final int[] BASE_DURABILITY = new int[] {13, 15, 16, 11};
+
+	// Protection values for all the slots.
+	// For reference, diamond uses 3 for boots, 6 for leggings, 8 for chestplate
+	// and 3 for helmet.
+	private static final int PROTECTION_BOOTS = 3;
+	private static final int PROTECTION_LEGGINGS = 6;
+	private static final int PROTECTION_CHESTPLATE = 8;
+	private static final int PROTECTION_HELMET = 3;
```

Comment:
I would inline these into the array creation, the constants dont add a lot. A comment could be put on each one saying what piece of armor it applies to.